lua/plugins/init.lua

```diff
@@ -96,13 +96,18 @@ local default_plugins = {
       vim.api.nvim_create_autocmd({ "BufRead" }, {
         group = vim.api.nvim_create_augroup("GitSignsLazyLoad", { clear = true }),
         callback = function()
-          vim.fn.system("git -C " .. '"' .. vim.fn.expand "%:p:h" .. '"' .. " rev-parse")
-          if vim.v.shell_error == 0 then
-            vim.api.nvim_del_augroup_by_name "GitSignsLazyLoad"
-            vim.schedule(function()
-              require("lazy").load { plugins = { "gitsigns.nvim" } }
-            end)
-          end
+          vim.fn.jobstart("git -C " .. '"' .. vim.fn.expand "%:p:h" .. '"' .. " rev-parse",
```

Comment:
This still requires a shell, move the command to a list-like instead of a string

Reply:
So the list works fine as well on `jobstart`, so I'll use it, as it means we can skip the shell entirely.